samples/Network/hdinsight-require-subscription-vnet/azurepolicy.parameters.json

```diff
@@ -0,0 +1,16 @@
+ {
+    "effect": {
+    "type": "String",
+    "metadata": {
+        "displayName": "Effect",
+        "description": "Enable or disable the execution of the policy"
+    },
+    "allowedValues": [
+        "Audit",
+        "Deny",
+        "Disabled"
+    ],
+    "defaultValue": "Audit"
```

Comment:
nit: Please indent consistently
(your favorite text editor can probably do this for you :])